Resources/Prototypes/Recipes/Lathes/security.yml

```diff
@@ -455,3 +455,75 @@
     Steel: 500
     Glass: 100
     Plastic: 200
+
+- type: latheRecipe
+  id: ShellShotgunUranium
+  result: ShellShotgunUranium
+  completetime: 2
+  materials:
+    Plastic: 20
+    Uranium: 2
+
+- type: latheRecipe
+  id: CartridgePistolUranium
+  result: CartridgePistolUranium
+  completetime: 2
+  materials:
+    Plastic: 10
+    Uranium: 1
+
+- type: latheRecipe
+  id: CartridgeMagnumUranium
+  result: CartridgeMagnumUranium
+  completetime: 2
+  materials:
+    Plastic: 20
+    Uranium: 2
+
+- type: latheRecipe
+  id: CartridgeLightRifleUranium
+  result: CartridgeLightRifleUranium
+  completetime: 2
+  materials:
+    Plastic: 20
+    Uranium: 2
+
+- type: latheRecipe
+  id: CartridgeRifleUranium
+  result: CartridgeRifleUranium
+  completetime: 2
+  materials:
+    Plastic: 15
+    Uranium: 1.5
```

Comment:
You can't have half a sheet of uranium, it's 1 or 2

Reply:
It did!